package.json

```diff
@@ -1,20 +1,20 @@
 {
-
-  "name": "fund-raising-project-client-side",
-  "version": "0.1.0",
-  "private": true,
-  "scripts": {
-    "dev": "next dev",
-    "build": "next build",
-    "start": "next start",
-    "lint": "next lint"
-  },
-  "dependencies": {
-    "eslint": "8.45.0",
-    "eslint-config-next": "13.4.12",
-    "next": "13.4.12",
-    "react": "18.2.0",
-    "react-dom": "18.2.0"
-  }
-
+    "name": "fund-raising-project-client-side",
+    "version": "0.1.0",
+    "private": true,
+    "scripts": {
+        "dev": "next dev",
+        "build": "next build",
+        "start": "next start",
+        "lint": "next lint"
+    },
+    "dependencies": {
+        "axios": "^1.4.0",
+        "eslint": "8.45.0",
+        "eslint-config-next": "13.4.12",
+        "next": "13.4.12",
+        "node-fetch": "^3.3.2",
```

Comment:
What do you need this package for? I don't see it being used anywhere

Reply:
node-fetch? Yeah, I've tried to install it but, after a while found out, that it is built in in Node.js 18. Fixed

Reply:
The `node-fetch` dependency is still there

Reply:
fixed